Frameworks/include/UIViewInternal.h

```diff
@@ -33,6 +33,27 @@
 @class WXFrameworkElement;
 @class WUXIPointerRoutedEventArgs;
 
+// Round subpixel values to be able to perform per-pixel UI placement/calculations
+inline float doPixelRound(float f) {
```

Comment:
>doPixelRound [](start = 13, length = 12)

Probably explains why I see Canvas.Left values of 101 when I expect 100. Experienced this during inset testing